samples/tfm_integration/CMakeLists.inc

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+project(NONE)
```

Comment:
Why is a `project` command needed here ?
It seems as the TF-M itself is build as `ExternalProject`.